bin/lighthouse/config.js

```diff
@@ -51,10 +51,28 @@ module.exports = {
       ],
       "td-has-header": [
         {
-          path: "/hc/en-us/subscriptions",
+          path: "/hc/:locale/subscriptions",
```

Comment:
Quick question: do we need this change due to the library updates?

Reply:
No. The URLs that are checked by the tool are [configurable by the user](https://github.com/zendesk/copenhagen_theme/blob/master/README.md#accessibility-testing). Somebody may put URLs with a different locale than `en-us`, so it is better to define the ignored rules with a placeholder. This one was added in #637 without the placeholder, probably by mistake